helloworld-samples/function-http-python/__main__.py

```diff
@@ -0,0 +1,23 @@
+import http.client
+import json
+
+
+def main(params):
+
+    url = "httpbin.org"
+    endpoint = "/get"
+    connection = http.client.HTTPSConnection(url)
+    connection.request("GET", endpoint)
```

Comment:
We could also extend this sample and conditionally use the users POST payload and send it to httbin.org/post, which will echo it.
What do you think?